### PR TITLE
feat: add observability v3.0 post

### PR DESCRIPTION
**Pull request by stijnmoreels**
Add blog post on the Observability v3.0 release.

Relates to #345

### CODE REVIEW

File: community/blog/2024-01 First Arcus (Observability) release of 2024 starts with a bang/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+The new release fixes this by interacting with the telemetry context in an immutable manner. Now the context can be used across logging so that the entire project logging can be streamlined. This is a great example of how practical usage of the Observability library results in better support and a more dev-friendly experience.
+
+## Conclusion
+Arcus Observability v3.0 is the first Arcus release of 2024 and immediately brings a whole lot of changes to the table. We expect soon other Arcus library releases with .NET 8 support.
+
+ Take a look at the [release notes](https://github.com/arcus-azure/arcus.observability/releases/tag/v3.0.0) and [feature documentation](https://observability.arcus-azure.net/) to learn more about Arcus Observability.
```
Comment (pim-simons) [inline community/blog/2024-01 First Arcus (Observability) release of 2024 starts with a bang/README.md:22]:
```suggestion
Take a look at the [release notes](https://github.com/arcus-azure/arcus.observability/releases/tag/v3.0.0) and [feature documentation](https://observability.arcus-azure.net/) to learn more about Arcus Observability.
```

**Review by pim-simons — APPROVED** [review]
💯

File: community/blog/2024-01 First Arcus (Observability) release of 2024 starts with a bang/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Removing deprecation
+The new release is a major release, which also brings some removals. All deprecated functionality that was previously released has been removed from this release. These are mostly logging extensions that were using incomplete or outdated parameters. The new available set of logging extensions is far simpler and easier to use.
+
+Some types were also removed, like the `DependencyMeasurement` which has been replaced by the `DurationMeasurement`. This was because we tracked not only the duration of dependency telemetry but also of requests, which made the old type illogical to use.
+
+## Reusable telemetry context
+There also exists a fix in this release, which is the result of frequently using Arcus in the wild. The Observability library provides many logging extensions, almost all of them with a telemetry context. In a real-life project scenario, you might want to reuse the telemetry context across your logging so that all logging is tagged with the same custom dimensions. The problem with earlier releases; was that this was not taken into account as the context was being used as a mutable set. Making it difficult to reuse.
```
Comment (fgheysels) [inline community/blog/2024-01 First Arcus (Observability) release of 2024 starts with a bang/README.md:15]: I don't see why we need a semicolon after 'earlier releases' ?

**Review by fgheysels — APPROVED** [review]
Great  article!
Just one question regarding an (unexpected) semicolon.

File: community/blog/2024-01 First Arcus (Observability) release of 2024 starts with a bang/README.md (hunk 1 above)
Comment (stijnmoreels) [reply community/blog/2024-01 First Arcus (Observability) release of 2024 starts with a bang/README.md:22]: Hehe, good catch. 😉

File: community/blog/2024-01 First Arcus (Observability) release of 2024 starts with a bang/README.md (hunk 2 above)
Comment (stijnmoreels) [reply community/blog/2024-01 First Arcus (Observability) release of 2024 starts with a bang/README.md:15]: Ah, yes, will remove that one. 👍

Comment (stijnmoreels) [reply community/blog/2024-01 First Arcus (Observability) release of 2024 starts with a bang/README.md]: Removed in https://github.com/arcus-azure/arcus/pull/346/commits/b9c3affd841090fb40788b217a64df9aac79b7b7

**Pull request merged**